remove the slow path(NCHW) for avg_pool3d

lly-zero-one opened the pull request:
Summary: Use the fast path for NCHW input tensor

Test Plan: run the pool unit tests

Differential Revision: D20522082

dskhudia reviewed (approved):
LGTM! Please make sure the tests pass.

lly-zero-one committed:
remove the slow path(NCHW) for avg_pool3d (#34994)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/34994

Use the fast path for NCHW input tensor

Test Plan: run the pool unit tests

Differential Revision: D20522082

fbshipit-source-id: 56fdb6dc095f40c9c410b74576dd7a214bc6ad56
